docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
-Sphinx==1.3
-sphinxcontrib-napoleon
+Sphinx>=1.7,<2
```

Comment:
in the future it would be great if we could find some way of versioning this w/ the requirements in setup.py

outside of the scope of this diff though

Reply:
yup agreed